app/views/practices/show/authenticated/_show_authenticated.html.erb

```diff
@@ -1,34 +1,60 @@
-        <%= render partial: "shared/messages", locals: {small_text: false} %>
-        <% unless @practice.published && @practice.approved %>
-          <div class="usa-alert usa-alert--warning">
-            <div class="usa-alert__body">
-              <h3 class="usa-alert__heading">This practice will not be shown in the marketplace because it is:</h3>
-              <% unless @practice.published %>
-                <div class="badge-container un-published-practice">
-                  Not Published
-                </div>
-              <% end %>
-              <% unless @practice.approved %>
-                <div class="badge-container un-approved-practice">
-                  Not Approved
-                </div>
-              <% end %>
+<% if @practice.retired %>
+  <div class="margin-bottom-3">
+    <section class="usa-section padding-y-0">
+      <div class="grid-container position-relative">
+        <div class="grid-row grid-gap">
+          <div id="practice_show" class="grid-col-12 overview-section practice-section">
+            <div class="usa-alert usa-alert--info">
+              <div class="usa-alert__body">
+                <h4 class="usa-alert__heading">Retired Practice</h4>
+                <p class="usa-alert__text">This practice is no longer being updated.
+                <% if @practice.retired_reason.present? %>
+                    <p>
+                    <%
+                      retired_reason_html = @practice.retired_reason.gsub("<p>", "").gsub("</p>", "")
+                    %>
+                    Reason for retirement: <%= retired_reason_html.html_safe %>
+                    </p>
+                 <% end %>
+                </p>
+              </div>
             </div>
           </div>
-        <% end %>
+        </div>
       </div>
-    </div>
+    </section>
   </div>
-</section>
-
+<% else %>
+  <section class="usa-section padding-y-0">
+    <div class="grid-container position-relative">
+      <div class="grid-row grid-gap">
+        <div id="practice_show" class="grid-col-12 overview-section practice-section">
+          <%= render partial: "shared/messages", locals: {small_text: false} %>
+          <% unless @practice.published && @practice.approved %>
+            <div class="usa-alert usa-alert--warning">
+              <div class="usa-alert__body">
+                <h3 class="usa-alert__heading">This practice will not be shown in the marketplace because it is:</h3>
+                <% unless @practice.published %>
+                  <div class="badge-container un-published-practice">
+                    Not Published
+                  </div>
+                <% end %>
+                <% unless @practice.approved %>
+                  <div class="badge-container un-approved-practice">
+                    Not Approved
+                  </div>
+                <% end %>
+              </div>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </section>
+<% end %>
```

Comment:
Can we just add the retired practice conditional to the original `section`? Is there a reason for splitting it into two parts?

Reply:
I did this refactor so that there is only one <section>